VMEncryption/ReleaseNotes.txt

```diff
@@ -1,3 +1,10 @@
+1.4.0.11
+- Shipping Azure Linux 3.0 support
+- MAJ_MIN bug fix
+
+1.4.0.10
+- <notes from release manager>
```

Comment:
@anshuljain26 did you release 1.4.0.10 by chance? Not seeing any code changes in dev branch; perhaps it was a re release from the RA-EV2 migration?